packages/protocol/contracts/common/GoldToken.sol

```diff
@@ -184,6 +173,10 @@ contract GoldToken is
    */
   function transferFrom(address from, address to, uint256 value) external returns (bool) {
     require(to != address(0), "transfer attempted to reserved address 0x0");
+    require(
```

Comment:
This will not disallow native transfer - we should do that by introducing receive function to CeloDistribution schedule

Reply:
According to the [docs](https://solidity-by-example.org/sending-ether/), a contract without fallback of receive function should not be able to receive native tokens.

I added a test case just to make sure.